app/services/sendspin_service.py

```
@@ -0,0 +1,937 @@
+        self._loop = loop
+        server_id = f"{self._server_name.lower().replace(' ', '-')}"
+        self._server = SendspinServer(
+            loop=loop,
+            server_id=server_id,
+            server_name=self._server_name,
+        )
+        self._unsubscribe_server = self._server.add_event_listener(self._on_server_event)
+
+        await self._server.start_server(
+            port=self._port,
+            host="0.0.0.0",
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)